src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -261,5 +261,31 @@ public async Task<ActionResult> CreateRelatedOrganisationAsync(Guid id, [FromBod
 
             return NoContent();
         }
+
+        [Authorize(Policy = PolicyName.CanAccessOrganisations)]
+        [HttpDelete]
+        [Route("{organisationId}/related-organisations/{relatedOrganisationId}")]
+        public async Task<ActionResult> DeleteRelatedOrganisationAsync(Guid organisationId, Guid relatedOrganisationId)
+        {
+            var organisation = await organisationRepository.GetByIdWithRelatedOrganisationsAsync(organisationId);
+
+            if (organisation is null)
+            {
+                return NotFound();
+            }
+
+            if (!organisation.RelatedOrganisations.Any(ro => ro.OrganisationId == relatedOrganisationId))
+            {
+                return BadRequest(new ErrorMessageViewModel(FormattableString.Invariant($"The referenced organisation {organisationId} has no relationship to {relatedOrganisationId}.")));
+            }
+
+            var relatedOrganisation = organisation.RelatedOrganisations.Where(ro => ro.OrganisationId == relatedOrganisationId).First();
```

Comment:
Semantically, I think `Single()` would fit better. I think we can also simplify by combining the `Where()` and `Single()` to:

```C#
var relatedOrganisation = organisation.RelatedOrganisations.Single(ro => ro.OrganisationId == relatedOrganisationId);
```